### ISSUE TITLE
The integration test suite fails if not all brokers join same node

### ISSUE THREAD

**Issue by menski** [zeebe-io/zeebe#2012]
After the migration to atomix we had to configure all nodes to join the same broker, as otherwise sometimes nodes where not visible to the gateway. We have 3 brokers and a gateway, where the following contact points are given

- `broker-0 -> null`
- `broker-1 -> broker-0`
- `broker-2 -> broker-1`
- `gateway -> broker-0`

Sometimes the waiting for the complete topology timed out as broker-2 was not visible to the gateway. In the logs seemed that the `broker-2` became visible to `broker-0` after the gateway joined.

This could just be a timing issues as the gateway might get the information of `broker-2` later by syncing its state. We have to verify that atomix guarantees that the cluster will converge into a stable state.

**Comment by menski** [comment]
@MiguelPires where you able to validate that this scenario is handled correctly by atomix? How long does it take until `broker-2` will be visible to the gateway?

**Comment by MiguelPires** [comment]
Hey @menski, I updated issue #2006 with some conclusions. In my experiment, the problems seem to arise when one node starts without its contact point starting first. Starting a node later than all the others wasn't a problem, when it joins the cluster the other node's listeners get MEMBER_ADDED events. I started node-2 a few seconds after all the others joined (including the gateway) and it didn't cause any problems. However, apparently if the initial contact point of a node isn't already up when that node starts, it won't try to contact it again which is a problem in the contact points are chained. I'm going to look into the test suite today
